servers.json

```diff
@@ -43,6 +43,13 @@
             "edenmine.eu"
         ]
     },
+    {
+        "id": "puritycube",
+        "name": "PurityCube Network",
+        "addresses": [
+            "mc.puritycube.eu"
```

Comment:
Lunar Client services automatically detect sub-domains, please remove the `mc.` prefix.